dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -220,17 +227,58 @@ def setUpClass(cls):
             },
         }
 
-    def check_continuous_properties(self, continuous_distribution_props):
+    def check_profile(self, profile, report=False):
```

Comment:
I think you just need to pass in a reference to the dict that is going to be used as the expected value since there are now two continuous distributions (weights and uniform). So maybe something like:
```suggestion
    def check_continuous_properties(self, continuous_distribution_props, expected_distribution_props):
```